gateway/platforms/servlet/src/main/java/io/apiman/gateway/platforms/servlet/connectors/ok/HttpURLConnectionImpl.java

```diff
@@ -121,7 +120,7 @@ public HttpURLConnectionImpl(URL url, OkHttpClient client) {
     // Calling disconnect() before a connection exists should have no effect.
```

Comment:
Update this comment perhaps?  No longer calling `disconnect()`

Reply:
Will fix